src/helm/benchmark/runner.py

```diff
@@ -86,13 +89,18 @@ def __init__(
         self.tokenizer_service = TokenizerService(self.executor.service, execution_spec.auth)
         self.metric_service = MetricService(self.executor.service, execution_spec.auth)
         self.skip_instances: bool = skip_instances
+        self.cache_instances: bool = cache_instances
+        self.cache_instances_only: bool = cache_instances_only
         self.skip_completed_runs: bool = skip_completed_runs
         self.exit_on_error: bool = exit_on_error
 
         ensure_directory_exists(output_path)
         # Decide where to save the raw data (e.g., "output/scenarios/mmlu").
         self.scenarios_path: str = os.path.join(output_path, "scenarios")
         ensure_directory_exists(self.scenarios_path)
+        # Decide where to save pre-processed instances
```

Comment:
Could you put an example here e.g. `output/scneario_instances/mmlu:subject=amatomy`

Also @percyliang could I get your opinion if this path is fine?

It _should_ be fairly easy to change this if we change our minds later.

Reply:
Path seems fine to me.  Not sure why "pre-processed" though.

Reply:
I guess in this context, "pre-processed" means "not the source CSV / JSON files?" I think it's fine to remove "pre-processed" from the comment.

Reply:
Done.